### PR TITLE
Add Big Query to tools page

### PR DESCRIPTION
**Pull request by inodb**
## Changes

- Add big query to tools page
- Migrate tools page from wordpress to static page

## Screenshot
Screen Shot 2022-07-21 at 6 24 41 PM

## Test Link
[Test Link](https://htan-portal-nextjs-git-add-bigquery-tool-htan.vercel.app/tools)

## TODO
- linebreaks don't look super pretty but can perhaps be addressed in another PR since it applies to all text on static pages

### CODE REVIEW

File: pages/static/tools.html [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+</p>
+<p>
+    <a href="https://isb-cgc.appspot.com/bq_meta_search/"
+        ><img
+            width="800"
+            src="https://user-images.githubusercontent.com/2837859/179309894-52004856-b61a-44ad-ac68-66539902b779.png"
+    /></a>
+</p>
+<p>
+    <a href="https://isb-cgc.appspot.com/bq_meta_search/"
```
Comment (clarisse-lau) [inline pages/static/tools.html:142]: Change to top level URL https://isb-cgc.appspot.com/

File: pages/static/tools.html [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+        >Explore HTAN BigQuery tables in ISB-CGC</a
+    >. Included are computational notebooks illustrating how to query and work
+    with HTAN data.
```
Comment (clarisse-lau) [inline pages/static/tools.html:145]: Add text: "To explore HTAN BigQuery tables in ISB-CGC, select **Launch** under `BigQuery Table Search` and **HTAN** from the `Program` dropdown. To access HTAN R and Python notebooks, select **GitHub** under `Notebooks` and navigate to **HTAN** folder."

**Comment by clarisse-lau** [comment]
Sorry for late revisions @inodb. We were thinking it would be better to provide explicit instructions on how to access the HTAN resources on ISB-CGC.

**Comment by inodb** [comment]
Thanks for reviewing @clarisse-lau ! Merging it now

**Comment by vthorsson** [comment]
Thanks @clarisse-lau and @inodb !

**Pull request merged**